openQA.spec

```diff
@@ -199,6 +199,16 @@ Supplements:    packageand(%name:postgresql-server)
 You only need this package if you have a local postgresql server
 next to the webui.
 
+%package single-instance
+Summary:        Convenience package for a single-instance setup
+Group:          Development/Tools/Other
+Requires:       %{name}-local-db
+Requires:       %{name}-worker
+Requires:       apache2
```

Comment:
This makes the package more convenient for Apache users and useless for everybody else. (I'm just saying; I don't care.)

Reply:
yes, this is true. But it is just providing a package for something that is prominent in our documentation the same and we provide files for apache, not for other services. I would like it if we also provide other variants, e.g. nginx along with any necessary nginx changes and then we can even just recommend/require "httpd" which is "provided" by both apache2 and nginx to be agnostic to the specific selection.